spyder/plugins/console/widgets/shell.py

```diff
@@ -631,9 +639,18 @@ class PythonShellWidget(TracebackLinksMixin, ShellBaseWidget,
     SEPARATOR = '%s##---(%s)---' % (os.linesep*2, time.ctime())
     go_to_error = Signal(str)
     
-    def __init__(self, parent, history_filename, profile=False, initial_message=None):
-        ShellBaseWidget.__init__(self, parent, history_filename, profile,
-                                 initial_message)
+    def __init__(self, parent, history_filename, profile=False,
+                 initial_message=None, default_foreground_color=None,
+                 error_foreground_color=None, traceback_foreground_color=None,
+                 prompt_foreground_color=None, background_color=None):
+        ShellBaseWidget.__init__(
+                        self, parent, history_filename, profile=profile,

```

Comment:
Hanging indent—is there a reason for the apparently non-standard positioning I'm missing?

Reply:
Just a personal preference. I normally left the indent in a way I find easier to read the begin and end of the parameters while using hanging indent (without using a specific number of spaces)